### DISCUSSION TITLE
如何解决'torch.distributed._shard._utils' 错误

### DISCUSSION THREAD

**Discussion by DerekAI** [zai-org/ChatGLM3#787] [Q&A]
尝试本地部署chatglm3-6b，在运行basic_demo时，出现如下错误：
RuntimeError: Failed to import transformers.generation.utils because of the following error (look up to see its traceback):
cannot import name 'narrow_tensor_by_index' from 'torch.distributed._shard._utils'

**Comment by zRzRzRzRzRzRzR** [comment]
能提供更多信息吗，比如是否在hf下载，硬件信息等
